### ISSUE TITLE
“Pods-RNPhotoEditor.debug.xcconfig” couldn’t be opened because there is no such file.

### ISSUE THREAD

**Issue by jrdesigner13** [prscX/react-native-photo-editor#132]
Hi,
Guys can someone help me?
I need to run on ios this old project that follows in this version:
Node version 10.16.3
"react-native": "0.57.8",
"react-native-photo-editor": "^1.0.6",

Android is running smoothly but on ios the error appears in this lib “Pods-RNPhotoEditor.debug.xcconfig” couldn’t be opened because there is no such file.
following the documentation it does not create the pods folder inside the lib.
node_modules/react-native-photo-editor/ios/ contains Pods folder. If does not exist please execute pod install command on node_modules/react-native-photo-editor/ios/, if any error => try pod repo update then pod install
Cocoapods is installed but inside that folder ios it does not create the pods folder
this Pods.xcodeproject file does not exist inside the ios folder
how can i solve this problem?

Regards

**Comment by hamzaouhssain1993** [comment]
did you follow the ios installation?
i advise you to upgrade to RN 60 before you get stuck to the old environment for a long time

**Comment by jrdesigner13** [comment]
Yes, I followed all the installation steps and unfortunately I cannot upgrade the version in this old project.
